src/handlers.js

```diff
@@ -93,33 +89,20 @@ export function taskParentGen(id)
 }
 export function eventDispatcher(requestedAppId,content)
 {
+    if(requestedAppId==='editor'){
```

Comment:
App Installation is done once, system wide - I think everytime we create a new file it should be installed as a seperate app with different content. You get what I'm trying to tell?

Reply:
I understand that. But that's a hack I tried to pass the content prop to the editor.